rfc/rfc018-oidc4vci-credential-issuance.md

```diff
@@ -0,0 +1,84 @@
+# RFC018 Verifiable Credential Issuance using OpenID
+
+|                           |           |
+|:--------------------------|:----------|
+| Nuts foundation           | R.G. Krul |
+| Request for Comments: 018 | Nedap     |
+
+## Verifiable Credential Issuance using OpenID
+
+### Abstract
+
+### Status of document
+
+This document describes a Nuts standards protocol.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+This specification describes how to apply OpenID for Verifiable Credential Issuance (OIDC4VCI) to issue Verifiable Credentials (VCs)
```

Comment:
```suggestion
This specification describes how to apply OpenID Connect for Verifiable Credential Issuance (OIDC4VCI) to issue Verifiable Credentials (VCs)
```

Reply:
I'm not sure where the `oidc4vci` comes from, but I propose to replace it with `OpenID4VCI` (so without `Connect` since that is not in the original spec, and same for `OpenID4VP`)
This is a lot easier to pronounce, and seems to be the preferred reference in the specs and it is also how they are referred to from the ARF